ner_v1/chatbot/entity_detection.py

```diff
@@ -387,6 +387,45 @@ def get_city(message, entity_name, structured_value, fallback_value, bot_message
     return None
 
 
+def get_name(message, entity_name, structured_value, fallback_value, bot_message):
+    """This functionality calls the NameDetector class to detect names
+
+    Attributes:
+        NOTE: Explained above
+
+    Output:
+        NOTE: Explained above
+
+    For Example:
+
+        message = 'My name is yash doshi'
+        entity_name = 'name'
+        structured_value = None
+        fallback_value = Guest
+        bot_message = None
+        output = get_city(message=message, entity_name=entity_name, structured_value=structured_value,
+                          fallback_value=fallback_value, bot_message=bot_message)
+        print output
+            //output without model
+            >> [{'detection': 'message', 'original_text': 'yash doshi',
+            'entity_value': {'first_name': yash, 'middle_name': None, 'last_name': doshi}}]
+    """
+    name_detection = NameDetector(entity_name=entity_name)
+    if structured_value:
+        entity_list, original_text_list = name_detection.detect_entity(text=structured_value, bot_message=bot_message)
+        if entity_list:
+            return output_entity_dict_list(entity_list, original_text_list, FROM_STRUCTURE_VALUE_VERIFIED)
+        else:
+            return output_entity_dict_value(structured_value, structured_value, FROM_STRUCTURE_VALUE_NOT_VERIFIED)
+    else:
+        entity_list, original_text_list = name_detection.detect_entity(text=message, bot_message=bot_message)
+        if entity_list:
+            return output_entity_dict_list(entity_list, original_text_list, FROM_MESSAGE)
+        elif fallback_value:
+            fallback_value = NameDetector.get_format_name(fallback_value.split())
```

Comment:
it's confusing.. why reassign to fallback value? Also using [0] and [1] hides whatever is return type..
instead use something on the lines of a,b = NameDetector.get_format_name(fallback_value.split())